### ISSUE TITLE
Bug: Contact form submit button displaying incorrect text

### ISSUE THREAD

**Issue by zeim839** [ufosc/Club_Website_2#185]
<img width="1213" alt="Screenshot 2023-11-01 at 15 51 45" src="https://github.com/ufosc/Club_Website_2/assets/50573884/8f631556-1e5f-4635-a2e4-5747ef6c9c8b">

Image: See [https://ufosc.org](https://ufosc.org)
(scroll to bottom)

The default value of the contact form 'submit' button varies between different browsers and devices, whereas we expect it to always be "submit". Sometimes, it will show up as "submit query", which is too large to fit within the button.

Solution: add a 'value' attribute to the input element in [views/index.ejs](https://github.com/ufosc/Club_Website_2/blob/main/views/index.ejs#L226) at line 226. For example:

```html
<input type="submit" name="submit" value="submit" class="contact__content__form-sumbit" style="background-color: #38F521;">
```

**Comment by ChowChunLeong** [comment]
this task can assign to me?

**Comment by zeim839** [comment]
@ChowChunLeong Sure. Let me know if you need any help.

**Comment by zeim839** [comment]
Implemented in #186 
Thanks @ChowChunLeong
